tests/unit/_experimental/test_plots.py

```diff
@@ -0,0 +1,1229 @@
+                            0.7100000000000001,
+                            0.7333333333333334,
+                            0.7566666666666667,
+                            0.78,
+                            0.8033333333333333,
+                            0.8266666666666667,
+                            0.8500000000000001,
+                            0.8733333333333334,
+                            0.8966666666666667,
+                            0.92,
+                            0.9433333333333334,
+                            0.9666666666666667,
+                        ],
+                        y=[
+                            0.6769230769230768,
+                            0.6769230769230768,
+                            0.6769230769230768,
+                            0.6769230769230768,
+                            0.6779661016949152,
+                            0.6779661016949152,
+                            0.6779661016949152,
+                            0.6779661016949152,
+                            0.6779661016949152,
+                            0.6315789473684209,
+                            0.6315789473684209,
+                            0.6315789473684209,
+                            0.6315789473684209,
+                            0.5555555555555555,
+                            0.5555555555555555,
+                            0.5555555555555555,
+                            0.5555555555555555,
+                            0.5283018867924528,
+                            0.5283018867924528,
+                            0.5283018867924528,
+                            0.5283018867924528,
+                            0.5,
+                            0.5,
+                            0.5,
+                            0.5,
+                            0.5,
+                            0.44,
+                            0.44,
+                            0.44,
+                            0.44,
+                            0.37499999999999994,
+                            0.37499999999999994,
+                            0.37499999999999994,
+                            0.37499999999999994,
+                            0.2926829268292683,
+                            0.2926829268292683,
+                            0.2926829268292683,
+                            0.2926829268292683,
+                            0.2926829268292683,
+                            0.15789473684210528,
+                            0.15789473684210528,
+                            0.15789473684210528,
+                        ],
+                        label="tps and fps and fns",
+                    ),
+                ],
+                x_config=None,
+                y_config=None,
+            ),
+            CurvePlot(
+                title="Precision vs. Recall",
+                x_label="Recall",
+                y_label="Precision",
+                curves=[
+                    Curve(
+                        x=[
```

Comment:
I tried running sklearn's `precision_recall_curve` and compared the results. 
It seems like precision and recall are incorrect @ threshold of `0.01`.  Could you please take a look at this @marko-polo-cheno ?

[pr_curve.py.zip](https://github.com/kolenaIO/kolena/files/11682132/pr_curve.py.zip)

Reply:
In the new PR, I think I've addressed all the comments. Most are due to a logic change. This case now matches perfectly, with the exception of the threshold @ -1, which I omit.

The two naive but massive issues are non-distinct thresholds with the `>=` vs `>` choice.